server/printing/printing_server.py

```diff
@@ -36,6 +38,11 @@ def SendRequestToPrinter(self, encoded_file, copies=1, options={}):
         command += "tmp.pdf"
         status = os.popen(command)
         os.remove("tmp.pdf")
+        printer = self.DeterminePrinterForJob(copies)
+        mock.get_printer(printer)
+        mock.add_left_printer_copies(self.pages[0])
+        mock.add_right_printer_copies(self.pages[1])
```

Comment:
This is really great! However, I don't see the spot where you declare `mock`

go ahead and declare it in the `serve` method with this line: `mock = <your class name>()`